docs/user-guide/security-features.md

```diff
@@ -1,152 +1,152 @@
 
-**Note:-**
+* **Global**
+* **Cluster**
+* **Environment**
+* **Application**
 
-It displays the “Vulnerability count and Severity Level” on a priority basis. And critical level has the highest priority, so it displays the critical level vulnerabilities and there counts if any application is having critical vulnerability in it.
+Policies are implemented in a hierarchical order, with the following sequence: Global, Cluster, Environment, and Application. Higher-level policies take precedence over lower-level policies, ensuring a systematic and structured enforcement of security measures.
 
-You can directly `Search` your application using the Search bar or you can `filter out` your requirement according to Severity, Clusters, and Environment.
-
-![](../.gitbook/assets/security-feature-securitytab-security-scan%20%282%29.png)
-
-Now if you click on the severity level of vulnerability it will show you the list of all vulnerabilities along with other details.
-
-![](../.gitbook/assets/security-feature-securitytab-list-vulnerability%20%282%29.png)
-
-## Security Policies:
+![](../.gitbook/assets/security-feature-global-security-policies.png)
 
-Users can define Security policies for their vulnerabilities under `Security Policies` Tab.
+**Some examples of how policies can be defined**
 
-Home Page-&gt; Security - &gt; Security Policies
+Users can block all the critical vulnerabilities and allow the moderate and low vulnerabilities or Users can block all vulnerabilities or users can block all vulnerabilities for one application and can block only Critical vulnerabilities for other applications.
 
-Policies can be defined to different levels-
+## Configure Global Security Policy
 
-* Global
-* Cluster
-* Environment
-* Application
+Within the Global Security Policies, there are two options available: Block and Allow.
+If critical severity levels are blocked in the Global Security Policy, the same blocking will be applied to the Cluster Security Policy. Similarly, if the global policy is modified to allow critical levels, it will also allow them in Cluster Security Policies. 
+However, users have the flexibility to explicitly modify these policies as desired.
 
-**Note:-**
+![](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/security-features/global-security-policy.jpg)
 
-Policies work in hierarchical order.
+## Configure Cluster Security Policy
 
-Order to be followed- First Global and second Cluster and so on as you can see the order of the options
 
-![](../.gitbook/assets/security-feature-global-security-policies.png)
+In Global Security Policies, there are two options: `Block` and `Allow`. Cluster Security Policies have an additional option called `Inherit`.
```

Comment:
In `Global Security Policies`, there are two options: `Block` and `Allow`. `Cluster Security Policies` have an additional option called `Inherit`.